README.md

```diff
@@ -1,7 +1,145 @@
-o-component-boilerplate [![Circle CI](https://circleci.com/gh/Financial-Times/o-component-boilerplate/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-component-boilerplate/tree/master)
+o-teaser [![Circle CI](https://circleci.com/gh/Financial-Times/o-teaser/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-teaser/tree/master)
 =================
 
-Origami component boilerplate.
+Article teaser component - shows a teaser of an article, requires a link through to the full article.
+
+- [Usage](#usage)
+	- [Markup](#markup)
+	- [Sass](#sass)
+- [Migration guide](#migration-guide)
+- [Contact](#contact)
+- [Licence](#licence)
+
+## Usage
+
+### Markup
+
+The basic markup structure for a teaser will look something like the below example:
+
+```html
+<div class="o-teaser o-teaser--small">
+	<div class="o-teaser__content">
+		<a href="#" class="o-teaser__tag">World</a>
+		<h2 class="o-teaser__heading"><a href="#">Japan sells negative yield 10-year bonds</a></h2>
+		<div class="o-teaser__timestamp">
+			<time data-o-component="o-date" class="o-date" datetime="2016-02-29T12:35:48Z">2016-02-29T12:35:48Z</time>
+		</div>
+	</div>
+</div>
+```
+
+Teasers support a wide array of [elements](#supported-elements) and can be customised using several [themes](#themes) and should be used as required. For a full list of examples including example markup, see the [o-teaser in the Registry](http://registry.origami.ft.com/components/o-teaser).
```

Comment:
Teasers support a wide array of [elements](#supported-elements) and can be customised using several [themes](#themes) and should be used as required. For a full list of examples including example markup, see ~~the~~ [o-teaser in the Registry](http://registry.origami.ft.com/components/o-teaser)